src/netex_france/exporter.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2017 Kisio Digital and/or its affiliates.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU Affero General Public License as published by the
+// Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+// details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>
+
+//! Exporter for Netex France profile
+use crate::{model::Model, Result};
+use chrono::prelude::*;
+use minidom::Element;
+use std::{
+    convert::AsRef,
+    fmt::{self, Display, Formatter},
+    fs::File,
+    path::Path,
+};
+
+const NETEX_FRANCE_STOPS_FILENAME: &'static str = "arrets.xml";
```

Comment:
clippy -> `&str`

Reply:
Yep, I just pushed that after I saw the CI complaining :smile: